src/components/CategoryFilter.js

```diff
@@ -7,12 +7,31 @@ const CategoryFilter = ({ handleFilterChange }) => {
     handleFilterChange(e.target.value);
   };
   return (
-    <select className="form-control" onChange={handleChange}>
-      <option key="default" value="All">All books</option>
-      {
-        categories.map((cat) => <option key={cat} value={cat}>{cat}</option>)
-      }
-    </select>
+    <nav className="mb-4">
+      <div className="row">
+        <div className="col-12 col-md-9">
+          <div className="row">
+            <div className="col-12 col-md-7 col-lg-6">
+              <h1 className="title-logo"><a className="a-logo" href="/">Bookstore CMS</a></h1>
+            </div>
+            <div className="col-12 col-md-2 col-lg-1 pt-2">
+              <a className="a-navbar" href="/">Books</a>
+            </div>
+            <div className="col-12 col-md-3 col-lg-2">
+              <select className="form-control" onChange={handleChange}>
+                <option key="default" value="All" selected>All books</option>
+                {
+                  categories.map((cat) => <option key={cat} value={cat}>{cat}</option>)
+                }
+              </select>
+            </div>
+          </div>
+        </div>
+        <div className="col-12 col-md-3 mt-1">
+          <i className="fas fa-user fa-lg fa-icon float-right" />
+        </div>
+      </div>
+    </nav>
```

Comment:
- [ ] Your margins and padding in the nav do not match the design, kindly do a thorough inspection on the zeplin design and fix this.
- [ ]  Your colors and fonts in the nav do not match the design, kindly do a thorough inspection on the zeplin design and fix this. The font is a free [google font](https://fonts.google.com/specimen/Montserrat?query=mon) that can be applied
- [ ]  Your nav is missing a label for the categories filter, kindly inspect the zeplin design and add this label as It appears in the design.

Reply:
Hi there:
-  I just use color cop to use the same colors. Could you show me what color do I need to improve?

Reply:
- Now I show you how I choose the blue color.
![image](https://user-images.githubusercontent.com/20563252/125808236-feda5591-76cf-4409-8398-384c5d7242dd.png)

Reply:
- Talking about margins and paddings on the navbar. I invite you to consider the next point, the project that appears on zeplin is for a static page. if I consider using exactly the same I need to choose screen size, if I do that my page will look bad for other screen sizes (including your own screen). Sometimes the elements between the navbar are gonna change the margin/padding because is what a responsive page does according to the screen size.
- Could you show me what exactly margin/padding do I need to improve?

Reply:
- I just take a minute but there we go, the "missing label" on my nav is exactly the select/option made in the previous milestone if you observe the navbar in my project you'll observe that categories are here. In this way, the user can select it.
![image](https://user-images.githubusercontent.com/20563252/125810415-f0923cff-ae10-4b1b-baa3-5b66c712b611.png)

Reply:
Hello, @vichuge I think you did great here. In honest assessment, you did this to the best of your ability. Good job with that. The colour topography is okay from my end and the fonts are just fine. I think this is a good implementation of the design.

Reply:
I consider don't use bootstrap would help to make the template more exactly, I mean doing from zero without any framework.